.github/workflows/integration-test.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         run: nix develop -c $SHELL -c "npm ci"
 
       - name: Run tests
-        run: nix develop -c $SHELL -c "npm t"
+        run: nix develop --override-input versions/holochain "github:holochain/holochain/holochain-0.3.0-beta-dev.48" -c $SHELL -c "npm t"
```

Comment:
Until Holonix is updated we need to test against a specific Holochain that is newer than the one in weekly.

As we work up to automatically testing new releases before releasing Holonix updates, we should find a way to make this simpler than changing the workflow file... Not sure how!